xla/service/gpu/runtime/nccl_collective_permute_thunk.cc

```diff
@@ -191,21 +164,21 @@ absl::Status NcclCollectivePermuteStartThunk::RunNcclCollective(
   const NcclP2PConfig::SourceTargetMapEntry source_target =
       NcclP2PConfig::GetSourceTarget(config_.id_to_source_target, current_id);
 
-  bool use_memcpy = is_local &&
-                    recv_value_map_.find(current_id) != recv_value_map_.end() &&
-                    send_value_map_.find(current_id) != send_value_map_.end() &&
+  bool use_memcpy = false;
+  use_memcpy = comm_wrapper.is_local &&
```

Comment:
NIT: why do we need a separate initialization and assignment. The two can be combined

Reply:
good catch, it's a peft-over from previous commit. deleted it.